packages/fiori/src/ShellBar.ts

```diff
@@ -139,15 +139,6 @@ const HANDLE_RESIZE_DEBOUNCE_RATE = 200; // ms
  * <li>product-switch</li>
  * </ul>
  *
- * <h3>CSS Shadow Parts</h3>
- *
- * <ui5-link target="_blank" href="https://developer.mozilla.org/en-US/docs/Web/CSS/::part">CSS Shadow Parts</ui5-link> allow developers to style elements inside the Shadow DOM.
- * <br>
- * The <code>ui5-shellbar</code> exposes the following CSS Shadow Parts:
- * <ul>
- * <li>root - Used to style the outermost wrapper of the <code>ui5-shellbar</code></li>
- * </ul>
- *
  * <h3>Keyboard Handling</h3>
  *
```

Comment:
- remove `@type`
- items slots to use interfaces instead of classes
- fix closeOverflow method tags and add explicit return type
- copilotDomRef / notificationsDomRef / overflowDomRef / profileDomRef / productSwitchDomRef remove type and add explicit return in ts
- _getAllItems fix usage of `@param`